setup.cfg

```diff
@@ -7,6 +7,8 @@ universal=1
 [flake8]
 exclude = setup.py,docs/*,examples/*,tests,graphene_django/debug/sql/*
 max-line-length = 120
+select = C,E,F,W,B,B950
+ignore = E501,W503
```

Comment:
We're trying to keep our "repo template" similar across the graphql-python ecosystem. Cross referencing with https://github.com/graphql-python/graphene/blob/master/setup.cfg and don't see these same config vars. What's the reason for adding?

Reply:
@mvanlonden I can remove these :)